reo/src/data_manager.py

```diff
@@ -105,6 +105,7 @@ def add_load(self, load):
         self.LoadProfile["resilience_check_flag"] = load.resilience_check_flag
         self.LoadProfile["bau_sustained_time_steps"] = load.bau_sustained_time_steps
         self.LoadProfile["annual_kwh"] = load.annual_kwh
+        self.LoadProfile["loads_kw"] = load.load_list
```

Comment:
@Bill-Becker this value is redundant with the one on line 103 above. Can the location that we are using "loads_kw" be switched to use "year_one_electric_load_series_kw"?

Reply:
I implemented this because @t-kwasnik told me it's needed for an API response's `inputs` to be used as a POST for another API run. Ted - if I misinterpreted or didn't give the full need, please let us know.